.env.example

```
@@ -18,7 +18,8 @@ NUXT_PUBLIC_POSTHOG_DISABLED=true
 # Maps and Auth
 GOOGLE_CLIENT_ID=GOOGLE_CLIENT_ID
 GOOGLE_CLIENT_SECRET=GOOGLE_CLIENT_SECRET
-GOOGLE_MAPS_API_KEY=GOOGLE_MAPS_API_KEY
+NUXT_PUBLIC_GOOGLE_MAPS_API_KEY=GOOGLE_MAPS_API_KEY
+NUXT_GOOGLE_MAPS_SERVER_API_KEY=AGOOGLE_MAPS_API_KEY
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NUXT_GOOGLE_MAPS_SERVER_API_KEY key should go before the NUXT_PUBLIC_GOOGLE_MAPS_API_KEY key

(UnorderedKey)